README.md

```diff
@@ -1,8 +1,33 @@
-Resources for the doc.opensuse.org website
-==========================================
+# doc.opensuse.org
 
-This repo holds the contents and the tools to create and update the static pages of doc.o.o.
-This public repo starts with a snapshot of the current files of the SUSE-internal repo.
+Built with [openSUSE Jekyll Theme](https://github.com/openSUSE/jekyll-theme).
 
-The apache config is not here.
-TODO: Provide info how to maintain redirects.
+Note this repo only contains the home page of the site. Actual documents are from:
+
+- https://github.com/SUSE/doc-sle
+- https://github.com/openSUSE/release-notes-openSUSE
+
+## Development
+
+### How to set up environment?
+
+```bash
+sudo zypper in ruby ruby-devel
+bundle install
```

Comment:
Typically it's very easy to avoid hardcoded version requirement by removing the BUNDLED WITH lines from Gemfile.lock, this should be done in the repo and not locally

Reply:
Should be fixed now. I removed `BUNDLED WITH`.

Reply:
* Thanks, build worked for me now.
* bundler suggested `bundle install --path vendor/bundle`, so maybe use that for the Readme as well, so people don't install everything into their system by default.